android/app/build.gradle

```diff
@@ -163,12 +169,17 @@ android {
 }
 
 dependencies {
-    compile project(':react-native-svg')
-    compile project(':react-native-vector-icons')
-    compile project(':react-native-maps')
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation project(':react-native-svg')
+    implementation project(':react-native-vector-icons')
+    implementation project(':react-native-maps')
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.android.support:appcompat-v7:$supportVersion"
+    implementation "com.facebook.react:react-native:$reactNativeVersion"  // From node_modules
+    implementation "com.android.support:multidex:1.0.2"
+    implementation "com.trinerdis.skypicker:react-native-native-modules:0.2-SNAPSHOT"
```

Comment:
should be replaced with `implementation project(':react-native-native-modules')`
so we can add it as npm dependency